### ISSUE TITLE
As a developer, I need to change back the metadata preview to previous status in which it displayed the full resource viewer

### ISSUE THREAD

**Issue by lilimelgar** [CLARIAH/wp5_mediasuite#421]
In a previous issue (#49) we requested that the pop-up window of the resource viewer in the result list only displayed the basic metadata, not the full metadata. 

![screen shot 2018-05-23 at 14 25 33](https://user-images.githubusercontent.com/8133228/40424213-394ce94e-5e95-11e8-9d36-a881c75f174f.png)

However, in further evaluations, because users cannot easily go back and forth between individual resources and result list without loosing their selections (non-saved bookmarks), they have requested to get more details (metadata) and the media in the pop-up window of the resource viewer. 

In summary: change the current content of the resource viewer (preview) in the result list (previous image) for the complete resource viewer (basic metadata, complete metadata, media, annotations, etc.)

**Comment by lilimelgar** [comment]
During summer school, for some users this seemed to be very important, while for others not so much. We close it for now, and may reopen it if it comes up in further evaluations.
